### PR TITLE
Remove quiver dependency, instead use hash functions provided by Dart

### PR DESCRIPTION
**Pull request by cenumi**

### CODE REVIEW

**Comment by shilangyu** [comment]
Bump, I think this is a beneficial change.

**Comment by devoncarew** [comment]
@kevmoo for review as he has commit access here (this change looks good to me)

File: pubspec.yaml [hunk 1]
```diff
@@ -5,10 +5,9 @@ description: Tuple data structure
 repository: https://github.com/google/tuple.dart
```
Comment (kevmoo) [inline pubspec.yaml:5]: ☝️  change the version to 2.0.1 and add a changelog entry!

**Review by kevmoo — CHANGES_REQUESTED** [review]

**Comment by devoncarew** [comment]
> change the version to 2.0.1 and add a changelog entry

I'll land this and update the changelog +version in a follow-up.

**Pull request merged**